libraries/x86/src/x86_interrupt.c

```diff
@@ -24,7 +24,7 @@ static uint8_t s_x86_interrupt_next_interrupt_id = 0;
 static uint8_t s_x86_interrupt_next_handler_id = 0;
 
 static Interrupt s_x86_interrupt_interrupts_map[NUM_X86_INTERRUPT_INTERRUPTS];
-static x86_interrupt_handler s_x86_interrupt_handlers[NUM_X86_INTERRUPT_HANDLERS];
+static x86InterruptHandler s_x86InterruptHandlers[NUM_X86_INTERRUPT_HANDLERS];
```

Comment:
The variable `s_x86InterruptHandlers` should be snake case?

Reply:
Done.